lib/seq-bitmap.js

```diff
@@ -0,0 +1,60 @@
+module.exports = SeqBitmap;
+
+function SeqBitmap() {
+  this.map = {};
+}
+
+/**
+ * Get the lowest available sequence index for the provided id and mark it as in use.
+ */
+SeqBitmap.prototype.requestNext = function(id) {
+  if (this.map[id] == null) {
```

Comment:
Is loose comparison intentional here? As far as I can see, `this.map` is of type `{[id: string]: undefined | number[]}` so maybe `this.map[id] === undefined` would be more relevant?